docs/userdocs/k8s/api/notebook.md

```diff
@@ -43,7 +43,7 @@ curl -X POST -H "Content-Type: application/json" -d '
     "resources": "cpu=1,memory=1.0Gi"
   }
 }
-' http://127.0.0.1:8080/api/v1/notebook
+' http://127.0.0.1:32080/api/v1/notebook
```

Comment:
Why change the service port?

Reply:
Because we deploy Traefik as default ingress controller in Submarine and it exposed with nodePort (32080)

Reply:
@lowc1012 Traefik port, can it be fixed?

Reply:
It is fixed for port 32080 if we use 'helm install' command. We can enable submarine expose another port by modified the[ parameter](https://github.com/apache/submarine/blob/master/helm-charts/submarine/charts/traefik/values.yaml#L210).